### ISSUE TITLE
vnu-jar.js

### ISSUE THREAD

**Issue by Tiliavir** [validator/validator#1974]
### Summary

My node skills are a bit rusty but something seems wrong ;-)

The vnu-jar.js in v25.12.18 is

```
module.exports = vnuJar;
module.exports.vnu = vnu;
```

shouldn't it be like the following?

```
module.exports = vnu;
module.exports.vnuJar = vnuJar;
```

The first tries to add a field to a string which seems wrong.

I tried to use it in a script like this:

```
'use strict';

import glob from "glob-all";
import vnu from 'vnu-jar';

await glob([
    'public/**/*.html',
    '!public/google*.html'],
  async (err, files) => {
    await vnu.check(files, {});
  }
);
```

calling it like this in my package.json script block:

`"validate": "node validate.mjs",`

### URL for document that reproduces the problem

-

### HTML/CSS/SVG snippet that reproduces the problem

```html
none required
```

### Checker error messages

```shell
module.exports.vnu = vnu;
                   ^

TypeError: Cannot create property 'vnu' on string '/home/.../node_modules/vnu-jar/build/dist/vnu.jar'
    at Object.<anonymous> (/home/.../node_modules/vnu-jar/vnu-jar.js:19:20)
    at Module._compile (node:internal/modules/cjs/loader:1358:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10)
    at Module.load (node:internal/modules/cjs/loader:1208:32)
    at Module._load (node:internal/modules/cjs/loader:1024:12)
    at Module.require (node:internal/modules/cjs/loader:1233:19)
    at require (node:internal/modules/helpers:179:18)
    at file:///home/.../validate.mjs:7:13
    at ModuleJob.run (node:internal/modules/esm/module_job:222:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
```

### Steps to reproduce

1.added in the description
2.
3.

### Expected behavior

as with my fix - no error in the execution

### Actual behavior

exception

**Comment by Tiliavir** [comment]
I am also a bit confused, that the published version on npm of 25.12.18 of the vnu-jar.js is not the one that is present on master nor the one on the feature branch sideshowbarker/npm-java-autoinstall.

edit:  Ah my fault - there as a new verison in the mean time - probably with the same issue:

```
const exported = new String(vnuJar);
exported.vnu = vnu;
module.exports = exported;
```

cant test it now though...

**Comment by sideshowbarker** [comment]
Thanks for reporting this.

> edit: Ah my fault - there as a new verison in the mean time - probably with the same issue:
> 
> ```
> const exported = new String(vnuJar);
> exported.vnu = vnu;
> module.exports = exported;
> ```

Yeah, that version — which I just now pushed —  is intended to fix the problem you reported. In my local environment at least, it definitely _does_ fix it. I mean, when you reported this, I could reproduce the same problem you’d reported. But with the above change, I can’t reproduce that problem any longer — and everything works as expected.

So, please do try it — when you can

**Comment by Tiliavir** [comment]
Works like a charm! Thanks a lot!
